ui/src/routes/conversations/[id]/+page.svelte

```diff
@@ -236,6 +271,51 @@
       })
     }
   }
+
+  let selectedMessageHash: string | null = null;
+    
+    function toggleMessageSelection(messageHash: string) {
+      selectedMessageHash = selectedMessageHash === messageHash ? null : messageHash;
+    }
+  
+    function unselectMessage() {
+      selectedMessageHash = null;
+    }
+  
+    function handleGlobalClick(event: MouseEvent) {
+      if (!event.target) return;
+      
+      const isInsideSelectedMessage = 
+        (event.target as HTMLElement).closest('.selected-message-container') ||
+        (event.target as HTMLElement).closest('.toolbar-container');
+      
+      if (!isInsideSelectedMessage) {
+        selectedMessageHash = null;
+      }
+    }
+  
+    function handleMessageClick(messageHash: string, event: MouseEvent) {
+      if(!isMobile()){
+        event.preventDefault();
+        toggleMessageSelection(messageHash);
+      }
+    }
+  
+    function handlePress(messageHash: string, event: PressCustomEvent) {
```

Comment:
```suggestion
    function handleMessagePress(messageHash: string, event: PressCustomEvent) {
```

good to have consistent naming for clarity

Reply:
Fixed the inconsistent naming issue.